src/store.js

```diff
@@ -508,6 +511,32 @@ class AsyncStorageStore {
     // We have finished the migration so we can set the storage version to the most recent one.
     this.updateStorageVersion();
   }
+
+  /**
+   * @param {string} pin
+   * @param {string} password
+   */
+  enableSafeBiometry(pin, password) {
+    // encrypt pin with password and save encrypted
+    const encryptedData = cryptoUtils.encryptData(pin, password);
+    this.setItem(PIN_BACKUP_KEY, encryptedData);
+    // Enable biometry
+    this.setItem(IS_BIOMETRY_ENABLED_KEY, true);
+  }
+
+  /**
+   * @param {string} password
+   * @returns {string}
```

Comment:
Can you add a description of what this is returning?

It's not intuitive that this method returns the PIN

Reply:
Done